README.md

```diff
@@ -108,34 +108,30 @@ their `onRecord` streams.
   });
 
 
-  // Will NOT print because FINER is too low level for `Logger.root`.
+  // Will NOT print because FINER is too low level for ([LOG1] & [ROOT]).
   log1.finer('LOG_01 FINER (X)');
 
-  // Will print twice ([LOG1] & [ROOT])
-  log1.fine('LOG_01 FINE (√√)');
```

Comment:
This one is incorrect.
It should print once by [LOG1] only as the root logger has higher level `WARNING`.
As per [Level](https://pub.dev/documentation/logging/latest/logging/Level-class.html) definiton, logging level should enable logging only for all levels above the level not below it.

However, the output shows that the root logger fires which is actually a bug https://github.com/dart-lang/logging/issues/145